hack/ci/Vagrantfile

```diff
@@ -7,7 +7,7 @@ Vagrant.configure("2") do |config|
   # so we specify the URL explicitly.
   # Mirrors can be found at here: https://admin.fedoraproject.org/mirrormanager/mirrors/Fedora/33/x86_64
```

Comment:
"33" in the comment lines should be updated to "34" as well

Reply:
Thank! I just want to trigger CI to determine whether the #2230  modification is in line with expectations.
Do I need to use this PR to replace #2230 and #2220 ?

Reply:
> Do I need to use this PR to replace #2230 and #2220 ?

Either is fine to me

Reply:
Addressed! Add Co-authored-by infos :smile: